mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -0,0 +1,50 @@
+package ch.epfl.sdp.mobile.ui.play
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.ExtendedFloatingActionButton
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.Add
+import ch.epfl.sdp.mobile.ui.PawniesColors
+import ch.epfl.sdp.mobile.ui.PawniesIcons
+
+/**
+ * Composable that composes the PlayScreen [TODO] Contains a new game button only, should be
```

Comment:
`[TODO]` ? What do you mean ?

Reply:
I agree it's not really clear clear. What I meant is :
`[TODO] Contains a new game button only, should be expanded to include history of matches`